### ISSUE TITLE
replace Twig_Extension with Twig\Extension\AbstractExtension

### ISSUE THREAD

**Issue by boris-brtan** [shopsys/shopsys#900]
### What is happening

after upgrade of vendor there were error #890 after some a while, fix was introduced for setting the problematic part to DEPRECATION warning 
```
DEPRECATION: Using the "Twig_Extension" class is deprecated since Twig version 2.7, use "Twig\Extension\AbstractExtension" instead. /var/www/html/vendor/twig/twig/lib/Twig/Extension.php:7
```  

### Expected result

To have resolved deprecation about `Twig_Extension`

**Comment by grossmannmartin** [comment]
https://www.tomasvotruba.cz/blog/2019/04/04/how-to-upgrade-twig-from-underscored-to-namespaces/

**Comment by TomasLudvik** [comment]
I have solved some deprecation in `tl-fix-deprecations` branch. This one has been solved in 112d4286e5e479dc2b5d336107937d03b76b3079

**Comment by pk16011990** [comment]
There is a problem with dependency... `PhpFileExtractor` uses `JMS\TranslationBundle\Translation\Extractor` wich has `\Twig_Node`. 

Package `JMS\TranslationBundle` must be updated before...

TIP for future volunteers/solvers: use rector v0.4.12 and remove phpstan from dependencis during refactoring... + be carefull for auto-replacing SSFW's twig extetensions in annotations, Rector replace them...

**Comment by henzigo** [comment]
.

**Comment by pesektomas** [comment]
I think so this has been solved in #1704.
